python/jupyter/src/widget.ts

```diff
@@ -48,6 +48,7 @@ class ChemiscopeBaseView extends DOMWidgetView {
             },
             this
         );
+        this._updatePythonSettings();
```

Comment:
Why this change?

Reply:
as per commit comment - otherwise on the python side the settings are uninitialized unless accessed from the notebook, so when you show and then chemiscope.save, you'll get empty settings otherwise.

Reply:
ok, my bad. I dug deeper and this was a red herring. problem was elsewhere, it's now fixed and this line has gone.

Reply:
code review done right, thank you!